ci/build.ps1

```diff
@@ -111,8 +111,8 @@ cmake --build . --config $cmakeBuildType -- $cmakeGeneratorArgs ; exitIfFailed
 .\bin\nvim -u NONE --headless -c 'exe !has(\"win32\").\"cq\"' ; exitIfFailed
 
 # Ensure Python is available.
-.\bin\nvim -u NONE -c 'exe !has("python")."cq"' ; exitIfFailed
-.\bin\nvim -u NONE -c 'exe !has("python3")."cq"' ; exitIfFailed
+.\bin\nvim -u NONE -c 'exe !has(\"python\").\"cq\"' ; exitIfFailed
+.\bin\nvim -u NONE -c 'exe !has(\"python3\").\"cq\"' ; exitIfFailed
```

Comment:
AFAIR the powershell escape char is backtick.

Reply:
This should be OK - IIRC I've tried the backtick also back with the has-win32 check/PR (recently merged).